samples/subsys/ipc/rpmsg_demo/src/rsc_table.cpp

```diff
@@ -0,0 +1,63 @@
+#include "rsc_table.hpp"
+#include <openamp/virtio.h>
+#include <stddef.h>
+#include <device.h>
+
+#define RSC_VDEV_FEATURE_NS (1) /* Support name service announcement */
+#define RSC_VDEV_NUM_VRINGS (2)
+
+#define VDEV0_VRING_BASE (DT_REG_ADDR(DT_NODELABEL(vdev0vring0)))
+#define VDEV1_VRING_BASE (DT_REG_ADDR(DT_NODELABEL(vdev0vring1)))
+
+#define VRING_ALIGN (0x1000U)
+#define RL_BUFFER_COUNT (256U)
+#define RESOURCE_TABLE_OFFSET (0xFF000)
```

Comment:
I don't see this being used in the `resource_table` sample, I know it's copied from the `mcuxpresso` example, but why is it needed?